gapic-common/lib/gapic/rest/grpc_transcoder/http_binding.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Gapic
+  module Rest
+    # @private
+    # Transcodes a proto request message into HTTP Rest call components
+    # using a configuration of bindings
```

Comment:
You should be able to omit this duplicated description because it's already present in the previous file (where this class is the "leaf").

Reply:
:+1: